src/frontend/js/widgets/Dashboard/components/LearnerDashboardSidebar/index.tsx

```diff
@@ -46,7 +47,7 @@ export const LearnerDashboardSidebar = (props: Partial<DashboardSidebarProps>) =
   return (
     <DashboardSidebar
       menuLinks={links}
-      header={intl.formatMessage(messages.header, { name: user?.username })}
+      header={intl.formatMessage(messages.header, { name: UserHelper.getName(user!) })}
```

Comment:
In `TeacherDashboardProfileSidebar/index.tsx` you used `header={user ? UserHelper.getName(user) : ''}`, why not here?

Reply:
Yup right let's use that here tto